src/components/FormInput/FormInput.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { FormContext } from '../FormContainer/FormContainer';
+import s from './FormInput.module.scss';
+
+function FormInput({ label, name, ...props }) {
```

Comment:
use 'useContext' hook instead of 'Consumer'

Reply:
left a todo in code